### DISCUSSION TITLE
Add R shortcuts of running code to or from the cursor

### DISCUSSION THREAD

**Discussion by vartimaios** [posit-dev/positron#4389] [Feedback] [answered]
R Studio has the shortcuts Ctrl + Alt + B to run the code from the beginning of the script to the cursor or Ctrl + Alt + E to run from the cursor to the end. Could we add these or equivalent shortcuts in Positron? They really increase efficiency when you want to make small changes in the code and re-run it.

**Comment by juliasilge** [comment, marked as answer]
Thanks for this suggestion! 🙌  I've opened https://github.com/posit-dev/positron/issues/4395 to track and prioritize this feature request.

**Reply by mchenpositron** [reply]
Really need this shortcut, too! It is very useful in RStudio.

**Reply by vartimaios** [reply]
Hi Julia! I wanted to ask, are there maybe any updates on the expected release of the shortcut?

**Reply by juliasilge** [reply]
No expected date currently, unfortunately. You can subscribe to #4395 for updates on when this gets prioritized.

**Comment by vartimaios** [comment]
Thank you Julia! Looking forward to the shortcut!
